Sources/DolbyIORTSUIKit/Private/ViewModels/ErrorViewModel.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ErrorViewModel.swift
+//
+    
+import DolbyIORTSCore
+import Foundation
+
+final class ErrorViewModel {
+
+    let titleText: String
+    let subtitleText: String?
+
+    init(titleText: String, subtitleText: String?) {
+        self.titleText = titleText
+        self.subtitleText = subtitleText
+    }
+    
+    init(error: Error) {
+        let titleAndSubtitle = Self.titleAndSubtitleForError(error)
+        self.titleText = titleAndSubtitle.title
```

Comment:
Better call the init() in line 13 above, to ensure the single same initializer is called every time in case of any change to the future.

Reply:
I used this initialiser for those times where we have an error object in hand, the assumption is leave the responsibility of checking the error types to populate the right message can be taken care by this error view model. Do you agree ?

Reply:
I mean.
```suggestion
convenience init(error: Error) {
        let titleAndSubtitle = Self.titleAndSubtitleForError(error)
        self.init(titleText: titleAndSubtitle.title, subtitleText: titleAndSubtitle.subtitle)
    }

```

Reply:
Yup, that makes sense to mark it a convenience init. Thanks, its updated now !!